pkgs/tools/misc/flameshot/default.nix

```diff
@@ -7,6 +7,8 @@
 , qtsvg
 , nix-update-script
 , fetchpatch
+, kguiaddons
+, withWaylandClipboard ? false
```

Comment:
Is there any reason we can't make it enabled by default? As mentioned in the flameshot issue, Arch had it [enabled by default](https://gitlab.archlinux.org/archlinux/packaging/packages/flameshot/-/blob/0c11b28c741ba7cdcc7af907f4b6fd6953e369dc/PKGBUILD#L26).

Reply:
Hmm 🤔 I was thinking that flameshot itself hasn't made it default yet. But then again, only downside I see is the extra dependency on kguiaddons.

I guess more people will run into the problem of Wayland clipboard not working than people worrying about the extra dependency.

I'll set it to true by default 👍